src/main/studio/ui/StudioPanel.java

```diff
@@ -1800,12 +1800,22 @@ public static void queryExecutionComplete(EditorTab editor, QueryResult queryRes
             try {
                 if (queryResult.isComplete()) {
                     JTabbedPane tabbedPane = panel.tabbedPane;
-                    TabPanel tab = new TabPanel(panel, queryResult);
-                    if (tabbedPane.getTabCount() >= CONFIG.getResultTabsCount()) {
-                        tabbedPane.remove(0);
+                    KTableModel model = KTableModel.getModel(queryResult.getResult());
+                    TabPanel modelTab = null;
+                    if (model != null) {
+                        modelTab = new TabPanel(panel, queryResult, model);
+                        modelTab.addInto(tabbedPane);
+                        modelTab.setToolTipText(editor.getServer().getConnectionString());
                     }
+                    TabPanel tab = new TabPanel(panel, queryResult, null);
```

Comment:
Console view is a great idea. However UX is confusing... If I drag&drop tabs, then there is no way to match console to visual (Table/List) tab. 

I prefer a concept that one result produces one tabs. And inside the tab there is a switcher to select either visual or console view.

Reply:
That would be a good future enhancement, for now the goal is to just restore the "lost" functionality so can we can do an internal release ASAP.
#78